it/uk/gov/hmrc/agentrelationships/stubs/DataStreamStub.scala

```diff
@@ -21,7 +23,7 @@ trait DataStreamStub extends Eventually {
              |}"""
         ))
       )
-    } (PatienceConfig(scaled(Span(2,Seconds)), scaled(Span(500,Millis))))
+    }
   }
 
   def verifyAuditRequestNotSent(event: AgentClientRelationshipEvent) = {
```

Comment:
`givenAuditConnector()` now stubs a response to the `/write/audit/merged` endpoint. 
Does `verifyAuditRequestNotSent` now need to verify `/write/audit/merged` was not called also?